linkcheck/updater.py

```diff
@@ -24,7 +24,9 @@
 
 # Use the Freecode submit file as source since that file gets updated
 # only when releasing a new version.
-UPDATE_URL = "https://raw.github.com/linkchecker/linkchecker/master/linkchecker.freecode"
+UPDATE_URL = (
+    "https://raw.github.com/linkchecker/linkchecker/master/linkchecker.freecode"
+)
```

Comment:
that seems unreasonable. use `#fmr off` `#fmt on` to ignore this block, i'd say.

Reply:
That whole file can go, it isn't actually used. I'll post after this one - I don't think we want to rebase this... (Could add the commit to this PR if preferred).

Reply:
FWIW I don't like many Black's decisions, but I like this one.